vscode-ext/src/extension.ts

```diff
@@ -177,52 +185,36 @@ class ConfigurationProvider implements vscode.DebugConfigurationProvider {
 			}
 		}
 
-		let libraryPath: string;
-		if (globalContext.extensionMode === vscode.ExtensionMode.Development) {
-			libraryPath = path.join(path.dirname(globalContext.extensionPath), "target", "debug");
+		let libraryPath: string, cliPath: string;
+
+		const mode = globalContext.extensionMode;
+		const extensionPath = globalContext.extensionPath;
+
+		if (mode === vscode.ExtensionMode.Development) {
+			const debugPath = path.join(path.dirname(extensionPath), "target", "debug");
+			libraryPath = debugPath;
+			cliPath = path.join(debugPath, "mirrord");
 		} else {
-			libraryPath = globalContext.extensionPath;
+			libraryPath = extensionPath;
+			cliPath = path.join(extensionPath, "mirrord");
 		}
+
 		let [environmentVariableName, libraryName] = LIBRARIES[os.platform()];
 		config.env ||= {};
 		config.env[environmentVariableName] = path.join(libraryPath, libraryName);
 
-		if (!await isTargetInFile()) { // If target wasn't specified in the config file, let user choose pod from dropdown
-			let podNamespace: string;
-			try {
-				podNamespace = await parseNamespace();
-			} catch (e: any) {
-				vscode.window.showErrorMessage('Failed to parse mirrord config file', e.message);
-				return;
-			}
-			let k8sApi = getK8sApi();
-			// Get pods from kubectl and let user select one to mirror
-			let pods: { response: any, body: V1PodList } = await k8sApi.listNamespacedPod(podNamespace);
-			let podNames = pods.body.items.map((pod) => pod.metadata!.name!);
-			let impersonatedPodName = '';
-			await vscode.window.showQuickPick(podNames, { placeHolder: 'Select pod to mirror' }).then(async podName => {
+		// If target wasn't specified in the config file, let user choose pod from dropdown			
+		if (!await isTargetInFile()) {
```

Comment:
Why'd you remove the namespace logic? If the user specified a namespace in mirrord.config, we want only targets from that namespace to be displayed.

Reply:
tbh I think better approach would be to just load the config file (so extension don't need to parse it)
but that might be an overkill right now

Reply:
Load it in the CLI? And then `mirrord ls` would be filtered without any additional args?

Reply:
or you can specify to override when using the cli, but yes

Reply:
Agreed that it's a better approach but overkill so lets stick with current implementation for now

Reply:
just making sure - right now the default is default which shouldn't be as it'd override the kubeconfig's default
if nothing in target file, don't set the flag.